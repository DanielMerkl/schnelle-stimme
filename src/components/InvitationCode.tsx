import React, { FC } from 'react';
import { Chip } from '@material-ui/core';
import styled from 'styled-components';

interface Props {
  code: number;
}

export const InvitationCode: FC<Props> = ({ code }) => (
  <StyledChip variant="outlined" label={`Code: ${code}`} />
);

const StyledChip = styled(Chip)`
  width: fit-content;
`;
