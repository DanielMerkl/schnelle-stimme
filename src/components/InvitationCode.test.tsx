import React from 'react';
import { render } from '@testing-library/react';

import { InvitationCode } from './InvitationCode';

describe('InvitationCode', () => {
  it('renders correctly', () => {
    const { container } = render(<InvitationCode code={12345} />);

    expect(container).toMatchSnapshot();
  });
});
