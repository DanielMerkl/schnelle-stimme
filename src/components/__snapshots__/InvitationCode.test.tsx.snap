// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InvitationCode renders correctly 1`] = `
<div>
  <div
    class="MuiChip-root sc-AxjAm gwKPmp MuiChip-outlined"
  >
    <span
      class="MuiChip-label"
    >
      Code: 12345
    </span>
  </div>
</div>
`;
